### ISSUE TITLE
CSS bugs

### ISSUE THREAD

**Issue by bd0bd** [backdrop-contrib/basis#55]
@wesruv 

There is a lot of css bugs when I look it in my Huawei MediaPad X1 in Chrome.
A lot!
Design is broken.

Please test.

**Comment by klonos** [comment]
Hello @bd0bd, Basis has been added in Backdrop core since 1.5.0. Are you seeing these bugs in Basis contrib or the one that comes with Backdrop core?

**Comment by bd0bd** [comment]
Hello @klonos 

BD 1.5.2 installed 1 day ago.

**Comment by klonos** [comment]
Yes, but the Basis theme you are seeing these bugs, is it the one that comes with Backdrop core or the one from contrib?

**Comment by bd0bd** [comment]
Sorry I do not know.

I have downloaded BD 1.5.2 from https://backdropcms.org/  DOWNLOAD BACKDROP
And then installed it on my server.

After I seen these bugs I was forced to switch to old reliable Bartik :smile:

**Comment by klonos** [comment]
Alright, if you have no "Basis" folder under your `/themes` folder, than it means that you are using the core Basis theme.

The reason I am asking is because you have filed this issue against Backdrop contrib (this was where Basis was developed before it was merged in core). Now, since Basis is included in core, you should file any bugs under Backdrop core and not Backdrop contrib.

No matter what, can you be more specific about the issue you are seeing? Can you provide some screenshots?
